src/main/resources/config/liquibase/changelog/20231101214350_added_entity_Synonym.xml

```diff
@@ -10,6 +10,11 @@
         Added the entity Synonym.
     -->
     <changeSet id="20231101214350-1" author="jhipster">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="synonym"/>
```

Comment:
I wasn't able to find that out. I can take some more time and try to get the `h2 console` working, which might provide some insight. I also tried running `clearChecksum` as suggested by https://stackoverflow.com/questions/66377049/table-already-exists-error-when-replacing-yaml-by-xml, but that didn't work either.

Reply:
It would be nice to find out why it happened. But I also know this post recommend always to add preConditions. The sequence of changeset doesn't seem to be honored always. https://stackoverflow.com/questions/61054021/best-practices-on-liquibase-preconditions